lib/SpeedTracker.js

```diff
@@ -315,7 +315,7 @@ SpeedTracker.prototype.runTest = function (profile, isScheduled) {
 
     if (!parameters.url) return Promise.reject('NO_URL')
 
-    let url = parameters.url
+    let url = parameters.url.trim().startsWith('http') ? parameters.url : Utils.decrypt(parameters.url, this.options.key)
```

Comment:
We're making the effort of trimming the string, which is good, but then we're not using that in the assignment if we're dealing with a plain text URL.

May I suggest something like:

```js
let url = parameters.url.trim()

if (url.startsWith('http')) {
  url = Utils.decrypt(parameters.url, this.options.key)
}
```